packages/flutter_test/lib/src/controller.dart

```diff
@@ -76,9 +104,6 @@ class SemanticsController {
   /// if no semantics are found or are not enabled.
   SemanticsNode find(FinderBase<Element> finder) {
     TestAsyncUtils.guardSync();
-    if (!_controller.binding.semanticsEnabled) {
-      throw StateError('Semantics are not enabled.');
-    }
```

Comment:
For my own understanding, why do we no longer want to enforce that semantics are enabled?

Reply:
Ah, we assert this already when you access `WidgetController.semantics`. All good.